format/view-spec.md

```diff
@@ -263,9 +263,6 @@ s3://bucket/warehouse/default.db/event_agg/metadata/00002-(uuid).metadata.json
   "format-version" : 1,
   "location" : "s3://bucket/warehouse/default.db/event_agg",
   "current-version-id" : 1,
-  "properties" : {
```

Comment:
shouldn't this be aligned with the first example where `properties` are specified?

Reply:
I thought it was intentional removal. To show some change in the new metadata. 

I added it back and updated the SQL.